core/dbt/events/base_types.py

```diff
@@ -167,3 +119,8 @@ class NoFile():
 # prevents an event from going to stdout
 class NoStdOut():
     pass
+
+
+@dataclass
+class NodeInfo():
+    node_info: Dict[str, Any] = field(default_factory=dict)
```

Comment:
Lets add a comment to this class to describe what it represents

Reply:
Done.